packages/webcert/src/store/api/apiMiddleware.ts

```diff
@@ -69,7 +85,9 @@ function createApiError(api: string, response: AxiosResponse | undefined, altMes
   if (!response) {
     return { api, errorCode: 'UNKNOWN_INTERNAL_PROBLEM', message: altMessage }
   }
-
+  if (response.data && response.data.INVALID_LAUNCHID) {
```

Comment:
Adjust based on the changes ask for in the Webcert PR. If you change in Webcert to use errorCode and message in the response, this code is redundant and the existing code on line 91 will work out of the box.